packages/rum/src/boot/recorderApi.spec.ts

```diff
@@ -158,6 +137,85 @@ describe('makeRecorderApi', () => {
     })
   })
 
+  describe('session renewal', () => {
+    let session: RumSessionMock
+    let lifeCycle: LifeCycle
+    beforeEach(() => {
+      session = createRumSessionMock()
+      setupBuilder.withSession(session)
+      ;({ lifeCycle } = setupBuilder.build())
+      rumInit(DEFAULT_INIT_CONFIGURATION)
+    })
+
+    describe('when an untracked session expires and the renewed session plan is REPLAY', () => {
```

Comment:
would it make sense to test all the possible combinations?
lite -> lite, replay -> replay, ...

Reply:
Yes, done